internal/pkg/externalplugins/merge/merge.go

```diff
@@ -22,15 +22,16 @@ const PluginName = "ti-community-merge"
 const githubUpdateCommitter = "web-flow"
 
 var (
-	addCanMergeLabelNotification   = "Can merge label has been added.  <details>Git tree hash: %s</details>"
+	addCanMergeLabelNotification = "This pull request has been accepted and is ready to merge. " +
+		"<details>Commit hash: %s</details>"
 	addCanMergeLabelNotificationRe = regexp.MustCompile(fmt.Sprintf(addCanMergeLabelNotification, "(.*)"))
-	configInfoStoreTreeHash        = `Guaranteed commits does not remove can merge label.`
+	configInfoStoreTreeHash        = `New commits will not remove the 'can-merge' label.`
```

Comment:
This is incorrect because only merging master commits using the GitHub button will not cause the can-merge label to be removed.

Reply:
I call this a guaranteed commit because GitHub adds a verified green logo for commit changes.

Reply:
Maybe `verified commits` is better?